src/services/account/services/accountApiService.js

```diff
@@ -64,6 +64,8 @@ class Accounts {
 
 const accountService = new Accounts({
 	paginate: false,
+	multi: true,
```

Comment:
i would suggest a `...serviceDefaults(options={multi, whitelist})` to be placed here that allows overriding the given props via options things like lean/paginate could be added there too.
discussion: object destructing or a schema-fabric/factory like structure...